_posts/2016-07-31-this-week-in-rustfest-8-tickets-and-talks.md

```diff
@@ -0,0 +1,29 @@
+
+This batch will exclusively sell the supporter and View Source combo tickets, regular tickets will be made available one week after, August 8th, at the same time. By buying a supporter ticket for 250€ (excl. VAT) you directly support the diversity efforts by the team, including directly increasing the amount of diversity and student tickets we can offer. If your company will be paying for your ticket, please consider these tickets – you'd help us and the community as a whole A LOT.
+
+Secondly View Source combo ticket, priced at 350€ (excl. VAT) for early-birds, allow you to visit [View Source Conf](https://viewsourceconf.org/berlin-2016/) in addition to RustFest. It's a 3-day single-track conference about the the future of the open web platform and related technologies that Mozilla organises earlier that same week in Berlin. Regular tickets will be available from August 8th at 125€ (excl. VAT).
+
+![](/assets/posts/transparenttickets.png)
+
+## More Talks announced
+
+The
+As already mentioned last week, we've closed the CfP and by now informed everyone about their participation. At the moment we are still in process of gathering all details and ensuring the logistics for everyone, but the basic talk schedule is clear now. Which allows us to announce the first set of  confirmed speakers and their [talks](/talks/): 🎉
+
+<div markdown="1" style="text-align: center; margin: 2.5em auto">
+  <span style="height:15vw; width:15vw; display: inline-block; vertical-align: middle" markdown="1">![](/assets/speakers/matthieu.jpg)</span> <span style="height:15vw; width:15vw; display: inline-block; vertical-align: middle" markdown="1">![](/assets/speakers/sharon.jpg)</span> <span style="height:15vw; width:15vw; display: inline-block; vertical-align: middle" markdown="1">![](/assets/speakers/kai.jpg)</span> <span style="height:15vw; width:15vw; display: inline-block; vertical-align: middle" markdown="1">![](/assets/speakers/ashley-williams.jpg)</span>
+</div>
+
+
+(From left to right) We are very happy **Matthieu Wipliez** will be sharing his experience and [Techniques for writing concurrent applications with asynchronous I/O](/talks/techniques-for-writing-concurrent-applications-with-asynchronous-i-o) in the web, before **Sharon Steed** will dig into team communication and collaboration in her talk [Empathetic communication: why vulnerability is the key to collaboration](/talks/#empathetic-communication-why-vulnerability-is-the-key-to-collaboration). While **Kai Michaelis** will show us [Panopticon - A Libre Cross-Platform Disassembler in Rust](/talks/#panopticon-a-libre-cross-platform-disassembler-in-rust) and share how Rust helped them find bugs and accelerate development, **Ashley Williams** will take us on the Journey of how she translated a declarative Ruby DSL to an ES6 parser in Rust in [Rust in Translation](/talks/#rust-in-translation).
```

Comment:
No problem that I just copy pasted the intro paragraph from last week? I think I should change it!